src/main/resources/org/jenkinsci/plugins/workflow/job/WorkflowRun/sidepanel.jelly

```diff
@@ -34,6 +34,37 @@
             <p:console-link/>
             <l:task icon="images/24x24/notepad.png" href="${buildUrl.baseUrl}/configure" title="${h.hasPermission(it,it.UPDATE)?'%Edit Build Information':'%View Build Information'}"/>
             <st:include page="delete.jelly"/>
+            <j:if test="${it.isBuilding() and h.hasPermission(it,it.CANCEL)}">
+                <j:choose>
+                    <j:when test="${it.hasStopBeenCalled() == false}">
+                        <l:task icon="images/24x24/edit-delete.png"
+                                href="${buildUrl.baseUrl}/stop"
+                                title="${%Stop Build}"
```

Comment:
> it definitely has to be more than just in the log

Not following. The option to `stop` was already in multiple places, just as for freestyle builds.

> we could just transform the stop button's actions

As noted below, I would rather keep `term` and `kill` as separate, explicit options, because they have more side effects than `stop`.